src/core/thread/mle_types.hpp

```diff
@@ -100,6 +100,9 @@ constexpr uint32_t kMaxResponseDelay               = 1000; ///< Max response del
 constexpr uint32_t kChildIdRequestTimeout          = 5000; ///< Max delay to rx a Child ID Request (in msec)
 constexpr uint32_t kLinkRequestTimeout             = 2000; ///< Max delay to rx a Link Accept
 constexpr uint8_t  kMulticastLinkRequestDelay      = 5;    ///< Max delay for sending a mcast Link Request (in sec)
+constexpr uint8_t kMaxCriticalTransmissionCount = 6; ///< Max number of times an critical MLE message may be transmitted
+constexpr uint32_t kMulticastTransmissionDelayMin = 4500; ///< Min delay for retransmitting a multicast packet (in msec)
+constexpr uint32_t kMulticastTransmissionDelayMax = 5500; ///< Max delay for retransmitting a multicast packet (in msec)
```

Comment:
Shall we define these constants as expressions of multicast retransmission delay?